source/assets/javascripts/common.js

```diff
@@ -4,13 +4,45 @@ import "bootstrap";
 import "details-element-polyfill";
 import "slick-carousel";
 import Headroom from "headroom.js";
-import AOS from "aos";
 import LazyLoad from "vanilla-lazyload";
 const Cookies = require('js-cookie');
 
 var lazyLoadInstance = new LazyLoad;
 
 $(function () {
+  // Animate objects only when they are visible
+  $.fn.isInViewport = function() {
+    var elementTop = $(this).offset().top;
+    var elementBottom = elementTop + $(this).outerHeight();
+
+    var viewportTop = $(window).scrollTop();
+    var viewportBottom = viewportTop + $(window).height();
+
+    return elementBottom > viewportTop && elementTop < viewportBottom;
+  };
+
+  $(window).on('resize scroll', function() {
+    $('.do-animation').each(function() {
+      if ($(this).isInViewport()) {
+        $(this).addClass('animated');
+      }
+    });
+    
+    var target = $('.count');
+    
+    if (target.data('start-counter') !== undefined) {
```

Comment:
@kennyadsl I moved the counter stats animation inside the  `$(window).on('resize scroll', function()` but I missed something because the counter text seems not able to read the value